autoload/javacomplete/classpath/gradle.vim

```diff
@@ -56,8 +56,8 @@ function! s:GenerateClassPath(path, gradle) abort
   let s:temporaryGradleFile = tempname()
   let s:gradleOutput = []
   let s:gradlePath = a:path
-  let gradle = has("win32") || has("win16") ? '.\gradlew.bat' : './gradlew'
-  if !executable(gradle)
+  let gradle = javacomplete#util#IsWindows() ? '.\gradlew.bat' : './gradlew'
+  if g:JavaComplete_EnableGradle && executable('gradle')
```

Comment:
but gralde is different with maven , we have two way build project,one it use the build script `./gradlew` or `.\gradlew.bat` in the root of current project ,but if we have install the application gradle, we can just use `gradle` command instead, and it will be faster